go/parquet/file/column_writer.go

```diff
@@ -597,6 +597,8 @@ func levelSliceOrNil(rep []int16, offset, batch int64) []int16 {
 	return rep[offset : batch+offset]
 }
 
+//lint:ignore U1000 maybeReplaceValidity
+//lint:ignore SA1019 ignore deprecated array.Interface
```

Comment:
instead of ignoring the deprecated message, might as well update this to be `arrow.Array` instead since you're in the file.